src/referencePackages/src/microsoft.win32.systemevents/4.7.0/microsoft.win32.systemevents.nuspec

```diff
@@ -31,11 +30,5 @@ When using NuGet 3.x this package requires at least version 3.4.</description>
       </group>
       <group targetFramework=".NETStandard2.0" />
     </dependencies>
-    <frameworkAssemblies>
-      <frameworkAssembly assemblyName="mscorlib" targetFramework=".NETFramework4.6.1" />
-      <frameworkAssembly assemblyName="mscorlib" targetFramework=".NETFramework4.7.2" />
-      <frameworkAssembly assemblyName="System" targetFramework=".NETFramework4.6.1" />
-      <frameworkAssembly assemblyName="System" targetFramework=".NETFramework4.7.2" />
```

Comment:
Did you intend to delete the 4.6.1 assemblies?

Reply:
No, will fix.

Reply:
Fixed.